python/setup.py

```diff
@@ -81,7 +81,7 @@ def config_cython():
             libraries = ['mxnet']
             # Default paths to libmxnet.so relative to the shared library file generated by cython.
             # These precede LD_LIBRARY_PATH.
-            extra_link_args = ["-Wl,-rpath=$ORIGIN/..:$ORIGIN/../../../lib:$ORIGIN/../../../build"]
+            extra_link_args = ["-Wl,-rpath,$ORIGIN/..:$ORIGIN/../../../lib:$ORIGIN/../../../build"]
```

Comment:
Do you know why the change is needed?

Reply:
There will be error when build with cython on MacOS
```
$ cd python
$ python3 setup.py build_ext --inplace --with-cython
...
ld: unknown option: -rpath=$ORIGIN/..:$ORIGIN/../../../lib:$ORIGIN/../../../build
clang: error: linker command failed with exit code 1 (use -v to see invocation)
error: command 'clang++' failed with exit status 1
```

Reply:
Nice catch. Can you change the github actions script to always test the cython build on os x?

Reply:
It's fine to do it here or in a separate PR